.github/ISSUE_TEMPLATE/1-bug-report.yml

```
@@ -74,7 +74,7 @@ body:
         Without this report we won't be able to fully evaluate this issue.
       placeholder: |
         The System Status Report is found in your WordPress admin under **WooCommerce > Status**. 
-        Please select “Get system report”, then “Copy for support”, and then paste it here.
+        Please select “Get system report”, then “Copy for GitHub", and then paste it here.
```

```suggestion
        Please select “Get system report”, then “Copy for GitHub”, and then paste it here.
```

Use curly quotes instead of straight.